src/main/java/org/opentripplanner/routing/api/request/preference/ItineraryFilterPreferences.java

```diff
@@ -48,6 +50,9 @@ private ItineraryFilterPreferences() {
         CostLinearFunction.of(Duration.ofMinutes(15), 1.5),
         0.4
       );
+    this.removeTransitWithHigherCostThanBestOnStreetOnly =
+      CostLinearFunction.of(Duration.ofMinutes(1), 1.3);
+    this.removeTransitWithMoreWalking = false;
```

Comment:
```suggestion
    this.removeTransitIfWalkingIsBetter = true;
```
Should this be default ON?

Reply:
This is how the filter was configured in the past. Some tests seem to assume that it is off by default.